pubspec.yaml

```diff
@@ -26,6 +26,8 @@ flutter:
       android:
         package: com.afur.flutterhtmltopdf
         pluginClass: FlutterHtmlToPdfPlugin
+      ios:
+        pluginClass: SwiftFlutterHtmlToPdfPlugin
```

Comment:
```suggestion
        pluginClass: FlutterHtmlToPdfPlugin
```

Reply:
I have tested this with the example project, and it works successfully

Reply:
Getting error

The plugin `newflutter` doesn't have a main class defined in /Users/apple/StudioProjects/newflutter/android/src/main/java/com/afur/flutterhtmltopdf/FlutterHtmlToPdfPlugin.java or /Users/apple/StudioProjects/newflutter/android/src/main/kotlin/com/afur/flutterhtmltopdf/FlutterHtmlToPdfPlugin.kt. This is likely to due to an incorrect `androidPackage: com.afur.flutterhtmltopdf` or `mainClass` entry in the plugin's pubspec.yaml.
If you are the author of this plugin, fix the `androidPackage` entry or move the main class to any of locations used above. Otherwise, please contact the author of this plugin and consider using a different plugin in the meanwhile.